### ISSUE TITLE
about section on more page is still in 'first person' from Eugen

### ISSUE THREAD

**Issue by spdegabrielle** [7guis/7guis#21]
https://7guis.github.io/7guis/more

> 7GUIs has been created as a spin-off of [my](https://eugenkiss.com/) master’s thesis [Comparison of Object-Oriented and Functional Programming for GUI Development](https://eugenkiss.com/projects/thesis.pdf) at the [Human-Computer Interaction group ](https://hci.uni-hannover.de/)of the Leibniz Universität Hannover in 2014. The GUI programming sphere has anything but stopped evolving since then. Yet, the holy grail appears to still be out of reach. I believe projects such as 7GUIs may help us find the right direction sooner. Four years after its inception I have written an article about 7GUIs: [Towards a Better GUI Programming Benchmark](https://hackernoon.com/towards-a-better-gui-programming-benchmark-397aca3542b8).

Suggested replacement text: 

> 7GUIs was been created as a spin-off of the master’s thesis *Comparison of Object-Oriented and Functional Programming for GUI Development* by Eugen Kiss at the [Human-Computer Interaction group ](https://hci.uni-hannover.de/)of the Leibniz Universität Hannover in 2014. Eugen wrote "The GUI programming sphere has anything but stopped evolving since then. Yet, the holy grail appears to still be out of reach. I believe projects such as 7GUIs may help us find the right direction sooner. Four years after its inception I have written an article about 7GUIs: [Towards a Better GUI Programming Benchmark](https://hackernoon.com/towards-a-better-gui-programming-benchmark-397aca3542b8)."

the website and the masters thesis are no longer available 

https://web.archive.org/web/20190708212055/https://eugenkiss.com/

**Comment by spdegabrielle** [comment]
[thesis.pdf](https://github.com/7guis/7guis/files/9207469/thesis.pdf)
 retrieved from wayback machine. https://web.archive.org/web/20180509155949/https://eugenkiss.com/projects/thesis.pdf

**Comment by spdegabrielle** [comment]
https://github.com/7guis/7guis/blob/7d260aee0ff065de00193e63953cd71f6f8c4e19/site/src/containers/More.js#L24

**Comment by dumblob** [comment]
Yes, I am out of time now, but please make a PR and include the copy of the thesis in the PR (along with the link to webarchive). Thanks!

**Comment by tbreuss** [comment]
Should be fixed with #26, too. 

@spdegabrielle Please review the PR, afterwards I will publish the changes.
